doc/design/staging-1.md

```diff
@@ -30,12 +30,22 @@ To test code changes to the web GUI, branch previews will also be pointed at the
 
 The production web GUI will track the `master` branch in `dandiarchive` and will automatically redeploy whenever `master` is changed.
 
-* A developer has a feature they want to add that impacts both the backend and the frontend.
+A developer has a feature they want to add that impacts both the backend and the frontend.
```

Comment:
in above text, not part of the PR, so can't comment there:
```
The staging web GUI will track the `master` branch in `dandiarchive` and will automatically redeploy whenever `master` is changed.
...
The production web GUI will track the `master` branch in `dandiarchive` and will automatically redeploy whenever `master` is changed.
```
doesn't sound right since they look identical to me and thus in effect we do not have "staging" (beyond state of PR) for "web GUI", is that the intention now (sorry if I didn't chime in time on that discussion). Or is that the current plan?

Reply:
if that it the plan, might be better to have a single sentence like
```
Both staging and production web GUI will track the `master` branch in `dandiarchive` and will automatically redeploy whenever `master` is changed.
```

Reply:
The next sentence for the staging GUI description clarifies:
```
The configuration variables for the staging Netlify deployment will point it at the staging server rather than the production server.
```
So there two permanent GUI servers that are running the exact same frontend code, but are pointing at different backends.

If you want to test frontend changes against staging, you would use a branch preview, which is also pointed at the staging API server.